ee_tests/cico_run_EE_tests_ts.sh

```diff
@@ -74,11 +74,12 @@ export GITHUB_USERNAME=$GITHUB_USERNAME
 export DEBUG="true"
 export QUICKSTART_NAME=$QUICKSTART_NAME
 export RELEASE_STRATEGY=$RELEASE_STRATEGY
+export RESET_ENVIRONMENT="true"
```

Comment:
We can't set it here because it would reset environment in all Jenkins jobs (i.e. logintest, smoketest, analyticstest, chetest).